src/styles/tablet/index.scss

```diff
@@ -0,0 +1,36 @@
+/**
+ * All tablet styles (strategic overrides of corresponding mobile/desktop styles)
+ */
+
+.navbar {
+    width: 80% !important;
```

Comment:
Do you need this important? I am very wary of using important unless it's _absolutely_ necessary

Reply:
resolved by refactoring